catalystneuro/nwb-conversion-tools
# added xml_file_path argument to neuroscope sorting; added docstring; …

Attn: @h-mayorquin 

Changes:
1. Added `xml_file_path` argument to NeuroscopeSortingInterface (depends on https://github.com/SpikeInterface/spikeextractors/pull/664)
2. Added docstring for NeuroscopeSortingInterface
3. Disabled waveform setting since, for NWB conversion purposes it's actually not working right now anyway. Once we get that upstream stuff done we can actually add a proper feature for it

## Motivation

Bug fix for when multiple `.xml` files exist in the same folder for NeuroscopeSorting data.

## How to test the behavior?
It's been on the to-do list for the NEO folks to add some NeuroscopeSorting data; I'll check in with them. Otherwise it's a bit hard to test this explicitly until we add that to GIN coverage.

## Checklist
